www/templates/html/MediaPlayer/Transcript.tpl.php

```diff
@@ -0,0 +1,79 @@
+<?php 
+    
+    $file = false;
+    try {
+        $vtt = new Captioning\Format\WebvttFile();
+        $tracks = $context->getTextTracks();
+        $file = $vtt->loadFromString(reset($tracks));
+    }catch(Exception $err){
+    }
+
+    $adapter = new \Conversio\Adapter\LanguageCode();
+    $options = new \Conversio\Adapter\Options\LanguageCodeOptions();
+    $options->setOutput('native');
+    $converter = new \Conversio\Conversion($adapter);
+    $converter->setAdapterOptions($options);
+?>
+
+<?php 
+    if($file){
```

Comment:
I was debating whether or not I should bring this up. In templates, we use the alternate syntax for control structures because it is easier to read when mixed with HTML. http://php.net/manual/en/control-structures.alternative-syntax.php

For example, this would be something like: 
```
<?php if($file): ?>
  ...
<?php endif; ?>
```

Along with this, avoid echoing html wherever possible and instead only echo php variables.